mlir/utils/jenkins/parameterSweeps.py

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env python3
```

Comment:
Also in this file, `miopen` refers to the MIOpen project except for MLIR tools `mlir-miopen-driver` and `miopen-gen` related variables.